modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -1327,6 +1327,8 @@ definition {
 				fieldType = "String",
 				isRequired = "true",
 				objectName = "CustomObject");
+
```

Comment:
Rename this task to "Given a published object with a field" using a commit called Wordsmith